biogrid/apps/biogrid-mvp/src/app/components/simulation-board.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import { Distance, ItemPosition } from '@biogrid/grid-simulator';
+import { ReactComponent as SmallBatterySvg } from '../../assets/icons/battery-small.svg';
+import { ReactComponent as LargeBatterySvg } from '../../assets/icons/battery-large.svg';
+import { ReactComponent as HouseSvg } from '../../assets/icons/house.svg';
+import { ReactComponent as GridSvg } from '../../assets/icons/grid.svg';
+import { ReactComponent as SolarPanelSvg } from '../../assets/icons/solar-panel.svg';
+import './simulate-board.css';
+
+export interface GridItemRet {
+  gridItemName: string;
+  relativePosition: ItemPosition;
+}
+
+export interface GridItemLines {
+  fromItem: string;
+  toItem: string;
+}
+
+interface SimBoardProps {
+  height: Distance;
+  width: Distance;
+  items: GridItemRet[];
+  lines: GridItemLines[];
+}
+
+interface ItemsByName {
+  [name: string]: ItemPosition;
+}
+
+const Grid = () => {
+  return (
+    <div className="grid-item">
+      <GridSvg />
+    </div>
+  );
+};
+const LargeBattery = () => {
+  return (
+    <div className="grid-item">
+      <LargeBatterySvg />
+    </div>
+  );
+};
+const SmallBattery = () => {
+  return (
+    <div className="grid-item">
+      <SmallBatterySvg />
+    </div>
+  );
+};
+const House = () => {
+  return (
+    <div className="grid-item">
+      <HouseSvg />
+    </div>
+  );
+};
+const SolarPanel = () => {
+  return (
+    <div className="grid-item">
+      <SolarPanelSvg />
+    </div>
+  );
+};
+
+export const SimulationBoard = (props: SimBoardProps) => {
+  const itemsByName: ItemsByName = props.items.reduce(
+    (map: ItemsByName, item) => {
+      map[item.gridItemName] = item.relativePosition;
+      return map;
+    },
+    {}
+  );
+  const kilometersToCSSWidth = (distance: Distance) => {
+    return `${Math.ceil((distance / props.width) * 100)}%`;
```

Comment:
is props.width the width of the entire screen? we probably never want something on the very edges of the screen and should make sure this map of the grid is padded on all sides

Reply:
Oh that is the size in of the grid in km, changed names to reflect.

Reply:
Hm not seeing this name change?